### PR TITLE
fix procedure syntax

### PR DESCRIPTION
**Pull request by xuwei-k**

### CODE REVIEW

File: src/reflect/scala/reflect/runtime/JavaUniverseForce.scala [hunk 1]
```diff
@@ -3,7 +3,7 @@ package scala.reflect
 package runtime
 
 trait JavaUniverseForce { self: runtime.JavaUniverse  =>
-  def force() {
+  def force(): Unit = {
```
Comment (martijnhoekstra) [inline src/reflect/scala/reflect/runtime/JavaUniverseForce.scala:6]: There is a comment here that it's generated code. Maybe this should be fixed in the generator? I have no idea *where* this code is generated.

Comment (hrhino) [reply src/reflect/scala/reflect/runtime/JavaUniverseForce.scala:6]: It's not _actually_ generated; the contents of this method are "verified" by some `run` partest (on my phone, but the file name is `run/t1234-universe-codegen` for a sufficiently large value of 1234). So this is apparently fine.

Comment (martijnhoekstra) [reply src/reflect/scala/reflect/runtime/JavaUniverseForce.scala:6]: Then the comment probably needs to be removed or improved, don't you think? Not sure if that's in scope of this PR under the boyscout rule (if @xuwei-k feels like including that) or that it should be a separate PR.

**Review by dwijnand — APPROVED** [review]
LGTM. I'm happy to defer the JavaUniverseForce.scala file header tweak to another PR.

**Comment by thinkerou** [comment]
I am beginner, recommend to use `Unit` when return null, right? thanks!

**Comment by lrytz** [comment]
@thinkerou please ask questions on https://gitter.im/scala/scala or https://users.scala-lang.org

**Comment by thinkerou** [comment]
@lrytz OK, thanks!

**Pull request merged**